charts/incubator/mealie/values.yaml

```diff
@@ -13,6 +13,11 @@ image:
   # -- image pull policy
   pullPolicy: IfNotPresent
 
+podSecurityContext:
+  runAsNonRoot: false
+  runAsUser: 0
+  runAsGroup: 0
```

Comment:
These are only complicating things.
It's only going to cause issues, because the default user is already user 0.

Reply:
Also: If you define things here, you also need to define them in iX_values (remember, ix_values REPLACES values.yaml).
But in these cases the correct solution is having no podsecuritycontext at all.

Reply:
You are correct, I put them to check the tests and forgot to remove,